app/views/spree/admin/orders/_invoice_table4.html.haml

```diff
@@ -37,8 +37,8 @@
           = item.display_amount_with_adjustments_and_with_taxes
     %tr
       %td
-        %strong= "#{t(:shipping)} "
-        = "( #{t(:invoice_shipping_type)} #{raw(@order.shipping_method.name)} )"
+        %strong= "#{@order.shipping_method.category} : "
+        = "(#{raw(@order.shipping_method.name)})"
```

Comment:
Does this have to be raw? I don't want enterprises to inject code via the shipping method name.

Reply:
No need for `raw` here.